src/mosaic-node-manager.cc

```diff
@@ -59,25 +59,24 @@ namespace ns3 {
     MosaicNodeManager::MosaicNodeManager() : m_ipAddressHelper("10.1.0.0", "255.255.0.0") {
     }
 
-    void MosaicNodeManager::Configure(MosaicNs3Server* serverPtr, CommunicationType commType=ClientServerChannelSpace::CommunicationType::DSRC) {
+    void MosaicNodeManager::Configure(MosaicNs3Server* serverPtr, CommunicationType commType) {
```

Comment:
Is there a reason this no longer has a default value?

Reply:
I put the default to the mosaic-nod-manager.h file